src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -371,6 +372,7 @@ public void periodic() {
 		launcherIsAtSpeed.setBoolean(isAtSpeed());
 		launcherAngleManual.setDouble(manualAngleSetpoint);
 		angleSetpointEntry.setDouble(angleSetpoint);
+		setLauncherSpeedEntry.setDouble(rpmSetpoint);
```

Comment:
Have we tested this? I'm not sure about the back-and-forth here between sets from Shuffleboard and sets from robot code.

Reply:
Also note that #110 also logs the setpoint, so the two PRs will likely conflict with each other.

Reply:
true, ill remove said changes later